core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -705,6 +707,22 @@ describe('Contributions and review component', () => {
         expect(component.activeExplorationId).toBeNull();
       }));
 
+    it('should be able to change language', fakeAsync(() => {
+      component.opportunitiesListRef = TestBed.inject(
+        OpportunitiesListComponent);
+      spyOn(component.opportunitiesListRef, 'onChangeLanguage')
+        .and.callFake(() => {
+          return;
+        });
+
+      component.onChangeLanguage('es');
+
+      expect(component.languageCode).toBe('es');
```

Comment:
Can we add a precheck for this? e.g. expect that the languageCode is different before onChangeLanguage is called

Reply:
Added!